TGEngine/gamecontent/Light.hpp

```diff
@@ -7,21 +7,37 @@
 namespace tge {
 	namespace gmc {
 
+		/*
+		 * This is the maximum count of lights used in the scene
+		 */
 		extern uint32_t lightCount;
 
+		/*
+		 * This holds the lights information
+		 *
+		 * <ul>
+		 * <li><strong class='atr'>pos</strong> is the position in the space</li>
+		 * <li><strong class='atr'>intensity</strong> is the intensity of the light</li>
+		 * <li><strong class='atr'>id</strong> is the id in the light array</li></ul>
+		 * <strong>This API is deprecated!</strong>
+		 */
 		class LightActor {
 
 		private:
 			glm::vec4 pos;
 			glm::vec4 intensity;
-			uint32_t id;
+			uint32_t  id;
```

Comment:
really had to add that extra space huh

Reply:
yes to properly line things up ... I would consider this a possible future standard ... because it makes it easier to fill the structs by just copy  pasting the parameters ... see the vulkan.hpp for further examples